src/ClaseServiciu/ServiciuScriereGeneric.java

```diff
@@ -0,0 +1,35 @@
+package ClaseServiciu;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class ServiciuScriereGeneric {
+    public <T, S> void scriereGenerica(T t, S s) {
```

Comment:
atat ServiciuScriereGeneric, cat si ServiciuCitire trebuia sa fie singleton, astfel ca nu voi putea puncta partea asta, iar in Main tu instantiezi claseie si cineva poate face oricate obiecte la un moment dat in Main de tipul acestor 2 servicii, cand ar fi trebuit sa apelezi metoda getInstance din servicii care returneaza de fiecare data un acelasi obiect, o sa te rog sa fixerzi pt 1p pt fixare comentarii